heartbeat/tomcat

```diff
@@ -56,7 +56,7 @@ else
   SU=su
 fi
 
-if ! ocf_is_true $OCF_RESKEY_force_script && which systemctl > /dev/null 2>&1; then
+if ! ocf_is_true $OCF_RESKEY_force_script && pidof systemd > /dev/null 2>&1; then
```

Comment:
Any reason why not checking for a known PID 1?  In general, it would likely be more reliable.